### PR TITLE
Alerting v2

### PR DESCRIPTION
**Pull request by rdubrock**
Adds an alerting tab to the plugin, which can create default alert rules and edit the generated rules. 

Screen Shot 2021-01-22 at 11 27 53 AM

Screen Shot 2021-01-21 at 3 21 59 PM

Screen Shot 2021-01-22 at 11 26 56 AM
Screen Shot 2021-01-22 at 11 26 22 AM

### CODE REVIEW

**Comment by rdubrock** [comment]
TBD: 
- Test styles with light mode
- Second pass at field validations in the alert rule form
- Update the alert expression to represent a percentage instead of an absolute probe success number

File: src/components/AlertRuleForm.tsx [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+      setIsOpen(false);
+      return Promise.resolve();
+    } catch (e) {
+      return Promise.reject(e.message ?? 'Something went wrong');
+    }
+  });
+
+  if (!defaultValues) {
+    return (
+      <Collapse label={rule.alert} isOpen={isOpen} onToggle={() => setIsOpen(!isOpen)} collapsible>
```
Comment (nathanrodman) [inline src/components/AlertRuleForm.tsx:179]: We're no longer using a <Collapse/> for the form, instead replacing it with a button. This should match

Comment (rdubrock) [reply src/components/AlertRuleForm.tsx:179]: Yes! Thanks a bunch

**Review by nathanrodman — APPROVED** [review]
LGTM :shipit: 

Great work, Russ!

**Comment by rdubrock** [comment]
Thanks a bunch for your help!

**Pull request merged**